app/process/maxspeed/maxspeed.lua

```diff
@@ -0,0 +1,434 @@
+  else
+    table.insert(tbl, -999)
+  end
+
+  local max = math.max(table.unpack(tbl))
+  if (not (tonumber(max) and -999)) then
+    print(max)
+    object.tags.maxspeed = max
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+end
+
+function DefineZones(object)
+  -- Translate maxspeed:type (and tagging variants) to maxspeed
+  if (object.tags["maxspeed:type"] == "DE:urban" or object.tags["source:maxspeed"] == "DE:urban") then
+    object.tags.maxspeed = "50"
+    object.tags.source_maxspeed = object.tags["source:maxspeed"]
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+
+  -- https://wiki.openstreetmap.org/wiki/DE:Key:maxspeed#Beispiele
+  if (object.tags["maxspeed:type"] == "DE:rural" or object.tags["source:maxspeed"] == "DE:rural") then
+    object.tags.maxspeed = "100"
+    object.tags.source_maxspeed = object.tags["source:maxspeed"]
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+
+  if (object.tags["maxspeed:type"] == "DE:zone30" or object.tags["source:maxspeed"] == "DE:zone:30" or
+      object.tags["source:maxspeed"] == "DE:zone30" or object.tags["zone:maxspeed"] == "DE:30" or
+      object.tags["zone:maxspeed"] == "30") then
+    object.tags.maxspeed = "30"
+    object.tags.source_maxspeed = object.tags["source:maxspeed"]
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+
+  if (object.tags["maxspeed:type"] == "DE:zone20" or object.tags["source:maxspeed"] == "DE:zone:20" or
+      object.tags["source:maxspeed"] == "DE:zone20" or object.tags["zone:maxspeed"] == "DE:20" or
+      object.tags["zone:maxspeed"] == "20")
+  then
+    object.tags.maxspeed = "20"
+    object.tags.source_maxspeed = object.tags["source:maxspeed"]
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+
+  -- Add note about conditional maxspeed
+  -- "maxspeed:conditional" — reichen wir durch; mehrwert ist noch fraglich; viele sonderfälle
+  if (object.tags["maxspeed:conditional"]) then
+    object.tags.maxspeed = "Angabe mit Einschränkungen"
+    object.tags._todo = false
+    object.tags._skip = false
+  end
+end
+
+function SortIntoTables(object)
+  if object.tags.maxspeed then
+    object.tags.is_present = true
+    if object.tags._skip == true then
+      -- maxspeed available but path type not correct -> _skip table
+      object.tags._todo = false
+    else
+      -- maxspeed available and path type is correct -> result table
+      object.tags._todo = false
+      object.tags._skip = false
+    end
+  else
+    object.tags.is_present = false
+    if object.tags._skip == true then
+      -- maxspeed not available and path type is not correct -> _skip table
+      object.tags._todo = false
+      object.tags._skip = true
+    else
+      -- maxspeed not available and path type is correct -> _todo table
+      object.tags._todo = true
+      object.tags._skip = false
+    end
+  end
+end
+
+function CheckSourceTagsValue(object)
```

Comment:
This function always goes into at least two of the else-cases, because `maxspeed` can only have on value